packages/core/onClickOutside/index.ts

```diff
@@ -55,6 +71,18 @@ export function onClickOutside<T extends OnClickOutsideOptions>(
   }
 
   let shouldListen = true
+  let stopDragListener: undefined | Function
+
+  const startDragListener = ({ x, y }: PointerEvent) => {
+    stopDragListener = useEventListener(window, 'pointermove', (e) => {
+      const dX = Math.abs(x - e.x)
```

Comment:
not sure if we should call `useEventListener` inside a function. 

This might lead to unexpected behaviour.